docs/content/releases/release-notes-1.7.0.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Release Notes 1.7.0"
+date: None
+draft: false
+toc: true
+description: Release information for NGINX Service Mesh, a configurable, low‑latency infrastructure layer designed to handle a high volume of network‑based interprocess communication among application infrastructure services using application programming interfaces (APIs).  Lists of new features and known issues are provided.
+weight: -1700
+categories: ["reference"]
+---
+
+## NGINX Service Mesh Version 1.7.0
+
+<!-- vale off -->
+
+These release notes provide general information and describe known issues for NGINX Service Mesh version 1.7.0, in the following categories:
+
+- [NGINX Service Mesh Version 1.7.0](#nginx-service-mesh-version-170)
+  - [Updates](#updates)
+  - [Known Issues](#known-issues)
+  - {{< link "/about/tech-specs" "Supported Versions" >}}
+
+<br/>
+<br/>
+<span id="170-updates"></a>
+
+### **Updates**
+
+NGINX Service Mesh 1.7.0 includes the following updates:
+<br/><br/>
+
+- `nginx-meshctl` command-line tool can now be downloaded from [Github](https://github.com/nginxinc/nginx-service-mesh/releases/latest).
+- NGINX Service Mesh can now integrate with the open source version of [NGINX Ingress Controller](https://github.com/nginxinc/kubernetes-ingress).
+- For easier integration with NGINX Ingress Controller, users can now [simply add a label]({{< ref "/tutorials/kic/deploy-with-kic.md" >}}) to their Ingress Controller PodSpec, and NGINX Service Mesh will automatically update the controller to integrate.
+- Sidecar and init containers are now built with ARM support.
+- OpenShift CSI Driver volume plugin has been renamed from `wlapi-mounter.spire.nginx.com` to `csi.spiffe.io`.
+
+{{< important >}}
+OpenShift users see the [upgrade guide]({{< ref "/guides/upgrade.md#upgrade-to-170-in-openshift" >}}) for instructions on how to upgrade to this release version.
+{{< /important >}}
+
+
+#### **Deprecation**
+
+- NGINX Ingress Controller annotations `nsm.nginx.com/enable-ingress` and `nsm.nginx.com/enable-egress` have been deprecated in favor of using labels.
+
```

Comment:
Do we repeatedly list all deprecations each release until removal or only list a deprecation once, in one release when it is declared?

Reply:
We've only listed them in the release when it's declared.